conan.lock

```diff
@@ -0,0 +1,90 @@
+        "viam-cpp-sdk/0.9.0#5b2598da2b28a40da22ea24967e64fce%1745266224.8786657",
+        "tensorflow-lite/2.15.0#d1d39a062bf00a61166d46a4537527fb%1724228962.993",
+        "ruy/cci.20231129#f23bfbab527048bfffba001f7a81236d%1702571818.886",
+        "re2/20230301#e8d673155a0494f27c7589b611e8350d%1709641579.83",
+        "pthreadpool/cci.20231129#7bea77db5af9a570f7510a92cada3065%1741701495.916",
+        "psimd/cci.20200517#83200a06ebb1ff39c5adff0d712c05fa%1700638971.11",
+        "protobuf/5.27.0#1fc80ae5667e6cd1d740ca84679b17d6%1742892879.34",
+        "pcre2/10.42#ac2227071f79a062167cf2fee7189d66%1727865720.942",
+        "openssl/3.4.1#85da54eb530cab53de0cc0ee9065d066%1741868597.788",
+        "nlohmann_json/3.11.3#45828be26eb619a2e04ca517bb7b828d%1701220705.259",
+        "lz4/1.9.4#652b313a0444c8b1d60d1bf9e95fb0a1%1721807480.477",
+        "libxcrypt/4.4.36#4b4e8f20794f1997dd59eeed0b7cdcfb%1696519152.422",
+        "libsystemd/255.10#0552f8de4d3eeaaefd87f6677dfbce1f%1723640836.911",
+        "libselinux/3.6#5a78ff6ae5034eeaac8da723361a8ce4%1717655459.344",
+        "libmount/2.39.2#64850679fa9d5ead878e5076b0cab2a8%1705591482.625",
+        "libcap/2.69#7ef2d60864c2e58e89db957be936dc49%1693427138.461",
+        "libbacktrace/cci.20210118#a7691bfccd8caaf66309df196790a5a1%1722218217.276",
+        "intel-neon2sse/cci.20210225#56e8b51d756e9ae2a612e3489039e07b%1701333748.561",
+        "grpc/1.69.0#937b22d3f20a2fdd2f2d278afadb8999%1742893931.477",
+        "gemmlowp/cci.20210928#28483fa833aa47549961bb9c5ab84bfa%1676224257.989",
+        "fxdiv/cci.20200417#0b3afe4c9d1b8d05f5f017984c8cb15a%1678543821.122",
+        "fp16/cci.20210320#34dbac7f6fa3dee68830028b53de6c84%1700638970.182",
+        "flatbuffers/23.5.26#606a38635994b68e946db3207b0159d7%1731000287.235",
+        "fft/cci.20061228#30ae3da3bd671aa271a4816b8e92dc33%1741856930.734",
+        "farmhash/cci.20190513#60980ffb85c63bbba6514c209f2ee1b8%1741856928.45",
+        "eigen/3.4.0#2e192482a8acff96fe34766adca2b24c%1680436083.8",
+        "cpuinfo/cci.20231129#15e94782b128bee8bfd047f6102a4d21%1716817358.974",
+        "c-ares/1.34.3#55aa867c731f23300cd544f6d90a794d%1731428505.375",
+        "bzip2/1.0.8#d00dac990f08d991998d624be81a9526%1724661266.998",
+        "boost/1.86.0#ce76e7477e466d7d8cbcf738c5d64175%1736197454.755",
+        "abseil/20240116.2#dfc978c35194fee939d236253044b967%1740151184.642"
+    ],
+    "build_requires": [
+        "zstd/1.5.5#1f239731dc45147c7fc2f54bfbde73df%1715599909.17",
+        "zlib/1.3.1#b8bc2603263cf7eccbd6e17e66b0ed76%1733936244.862",
+        "xz_utils/5.4.5#b885d1d79c9d30cff3803f7f551dbe66%1724318972.064",
+        "re2/20230301#e8d673155a0494f27c7589b611e8350d%1709641579.83",
+        "protobuf/5.27.0#1fc80ae5667e6cd1d740ca84679b17d6%1742892879.34",
+        "pkgconf/2.2.0#6462942a22803086372db44689ba825f%1713364853.749",
+        "pkgconf/2.1.0#27f44583701117b571307cf5b5fe5605%1701537936.436",
```

Comment:
Per the discussion in https://github.com/viamrobotics/viam-cpp-sdk/pull/420, multiple versions of the same package are okay in the "build_requires" section because these are things used to build tools and tests, not things to build the final executable. The "requires" section, above, is the stuff that actually goes into the final executable, and each package up there should have one and only one version.